src/components/editor/InputLiteral.jsx

```diff
@@ -169,7 +146,9 @@ export class InputLiteral extends Component {
 
   makeAddedList = () => {
     let formInfo = this.props.formData
-    if (formInfo == undefined) return
+    // console.log(`makeAddedList ${this.props.propertyTemplate.propertyLabel}`)
+    // console.warn(formInfo)
+    if (formInfo == undefined || formInfo.items == undefined) return
```

Comment:
this looks funny to me ... would we want the "return" on the next line, along with indenting the following lines appropriately?   Not a biggie, just asking.

Reply:
This just follows the syntax for a `if` statement one-liner; changed to separate line to improve readability.